README.md

```diff
@@ -7,13 +7,13 @@
 |Mar 27th, 2020. OAM v1.0.0-alpha.2 is **RELEASED**! The new spec is highly extensible and native to Kubernetes runtime. Check the spec and [What's new in OAM v1alpha2](https://speakerdeck.com/resouer/whats-new-in-oam-v1alpha2-spec) for more detail!|
 |Mar 26th, 2020. A proof-of-concept project named AWS ECS for OAM is published! Check [the AWS Labs repo ](https://github.com/awslabs/amazon-ecs-for-open-application-model) and have fun with developer centric experience with OAM + Fargate!|
 
-Open Application Model is a specification for building cloud native applications.
+Open Application Model is a specification for defining cloud native applications and higher level abstractions.
```

Comment:
what does "higher-level abstractions" refer to?

Reply:
For example, ContainerizedWorkload is a higher level abstraction for Deployment. Any better wording?

Reply:
It seems to me that an application is already a higher level  concept than deployment, I thought the "higher-level abstraction" is even higher than an application.

Reply:
Then I will keep this sentence unchanged, this part is not a big deal.